src/oss/confluent.sh

```diff
@@ -1741,6 +1741,13 @@ invalid_requirement() {
 
 requirements
 
+cat <<EOF
```

Comment:
For your consideration, do we want a banner?  e.g. `=============`

Reply:
I think it looks good and clear as-is (without banner).